learning/katas/go/io/textio/read/testdata/path.go

```diff
@@ -0,0 +1,40 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testdata
+
+import (
+	"path/filepath"
+	"runtime"
+)
+
+var (
+	// dataDir is the absolute path to the directory of this package
+	dataDir string
+)
+
+func init() {
+	_, runtimeFilePath, _, _ := runtime.Caller(0)
```

Comment:
Clever!

Reply:
I learned it from this project https://github.com/grpc/grpc-go/blob/master/testdata/testdata.go.  It definitely makes things easier.